INSTALLATION.md

```diff
@@ -84,3 +85,47 @@ In the file add the line
 Replace `<username>` with the output of `whoami` and `/path/to/nginx` with the output of `which nginx`. Further information is available via [StackOverflow](http://askubuntu.com/questions/159007/how-do-i-run-specific-sudo-commands-without-a-password).
 
 You will also need to `chown` the Nginx error log so it is accessible by your user. You can see the location of this log file when you run `nginx -V` as the `--error-log-path` configure argument's parameter.
+
+### Use Nginx with Docker
+By aliasing the `nginx` command to start a Docker container, it's also possible to run proxrox without having Nginx
+directly installed.
+
+Create a Bash script and make it executable, with the following contents:
+
+```shell
+#!/bin/bash
+
+configpath=$2
+tmpdir=$4
+echo "===> configpath: ${configpath} and tmpdir: ${tmpdir}"
+
+del_stopped(){
+  local name=$1
+  local state=$(docker inspect --format "{{.State.Running}}" $name 2>/dev/null)
+
+  if [[ "$state" == "true" ]]; then
+    docker stop $name
+    docker rm $name
+  elif [[ "$state" == "false" ]]; then
+    docker rm $name
+  fi
+}
+
+del_stopped proxrox-nginx
+
+docker run -d \
+         --restart always \
+         -v "${configpath}:/etc/nginx/nginx.conf" \
```

Comment:
How does this work with the `root` configuration option? As far as I can tell it does not?

Might be acceptable to some users if we document this as a limitation/caveat.

https://github.com/bripkens/proxrox/blob/master/CONFIGURATION.md#root

Reply:
Should work as well, but then a volume mapping needs to be added so that the `root` path can be found. Will describe that as well.